example/src/components/ComponentDocTemplate.js

```diff
@@ -197,18 +212,20 @@ function ComopnentDocTemplate(props) {
                 <CopyLinkContainer onClick={copyToClipboard}>
                     <img src={copyLink} alt={"Click to copy URL for this component."} />
                 </CopyLinkContainer>
+                <HiddenParagraph id={'chuckles'}>Copied to clipboard!</HiddenParagraph>
```

Comment:
This is a code smell for me, meaning something feels off but I'm not sure what. I feel like I never see `id` used in React. We may want to look in to `useRef` instead.